e2e/transactions/SendNft.yaml

```diff
@@ -33,13 +33,29 @@ tags:
           maxRetries: 3
           commands:
             - swipe:
+                from:
+                  id: wallet-screen
                 direction: UP
             - assertVisible: '.*Rainbow Pooly.*'
 
-      - tapOn: '.*Rainbow Pooly.*'
+      - retry:
```

Comment:
as you progress through this flow we keep expanding views that open downward, sometimes it doesn't scroll far enough. the retry works great here bc if it fails it just continues from the same screen position as the failure and scrolls down more which seems to always fix the issue.

Reply:
sure! as long as it works! changed it

Reply:
Oups looks like it doesnt work on ios, I only tested the fix on android. Can go back to yours if you want, unless you wanna debug it 😅

Reply:
reverted ! 💯